### PR TITLE
Update charging station mapping

### PR DESCRIPTION
**Pull request by francois2metz**
I updated the mapping with the capacity and the type of the sockets

### CODE REVIEW

File: mappings/charging_station.yml [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 mapping:
-  covered: covered|bool
   opening_hours: opening_hours
   parking_fee: "parking:fee"
-  start_date: start_date
   owner: owner
   operator: operator
   website: website
-  chademo: "socket:chademo|bool"
+  capacity: capacity
```
Comment (Phyks) [inline mappings/charging_station.yml]: capacity|int ? (To ensure an int value in the JSON output)

File: mappings/charging_station.yml [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+  chademo: "socket:chademo"
```
Comment (Phyks) [inline mappings/charging_station.yml]: Oh, these are number of sockets, right? Then, maybe "socket:chademo|int" etc for the same reason?

File: mappings/charging_station.yml (hunk 1 above)
Comment (francois2metz) [reply mappings/charging_station.yml]: fixed

File: mappings/charging_station.yml (hunk 2 above)
Comment (francois2metz) [reply mappings/charging_station.yml]: fixed

**Pull request merged**